### ISSUE TITLE
Error display map location in profile visitor: TypeError: map.viewAB is undefined

### ISSUE THREAD

**Issue by samtux** [piwik/piwik#8997]
When I try to display the map location in the visitor profile reports I get the following error:

```
TypeError: map.viewAB is undefined index.php:2583:475
```

**Comment by mattab** [comment]
Hi @samtux 
please try with latest 2.15.0 RC http://piwik.org/changelog/piwik-2-15-0/

**Comment by samtux** [comment]
@mattab, the error persists. Additionally with this version 2.15 RC, does not allow to display the map of visits on the board, because that always displays the message "No data for this report."

My configuration of Piwik is with Centos 6.2, Nginx and PHP-FPM.

**Comment by mattab** [comment]
@samtux could you please send us an email with your URL, login and password, so we could take  alook?

**Comment by samtux** [comment]
@mattab, thank you very much for the help. I could not find your email contact, you can supply it?

Thanks.

**Comment by mattab** [comment]
Hi @samtux please try again with our latest beta 2.16.0 which hopefully will fix this issue!

**Comment by gregr77** [comment]
Hi guys. I use recent 2.16.1 version of piwik (installed yesterday - 8 of June 2016) and when I click on a map in visitor profile I am getting "the same" error except on different line: TypeError: map.viewAB is undefined /index.php?module=Proxy&action=getCoreJs&cb=... Line: 2798
I can provide any additional information you need. We just started playing with piwik and the only "additional" module we use is PHP GeoIP.
Thanks,
Greg

**Comment by mattab** [comment]
Hi @gregr77 

Thanks for the report. Do you get this error in both firefox and chrome? could you post a screenshot?

**Comment by gregr77** [comment]
Hi guys,

Yes I got an error using all browsers but I found the reason looking at
another issue and fixed it later. The problem was not browser but server
settings related. Some mime types were not defined in IIS where piwik is
hosted and it resulted in maps not displayed. I believe it was .svg
extension required but I added mapping to few of them (.svg, .xht, .xhtml
and may be others) at the same time so I am not 100% sure.

I think

Best Regards,
Greg

Greg Ryjikh,
Senior Software Engineer
DDSN Net Pty Ltd
ABN 67 165 575 039

E: gregr@ddsn.com
W: http://www.ddsn.com
W: http://www.cm3cms.com
W: http://extranet.ddsn.com

Melbourne, Australia
Level 4, 574 St Kilda Rd, Melbourne, VIC 3004
P: 1300 656 412
P: +61 3 9533 8800
F: +61 3 9533 8899

This email is sent commercial in confidence
